bcda/api/requests.go

```diff
@@ -538,8 +538,16 @@ func (h *Handler) bulkRequest(w http.ResponseWriter, r *http.Request, reqType se
 		logger.Info("job id created")
 	}
 
+	jobData := models.JobEnqueueArgs{
```

Comment:
since GetPatient now requires the JobEnqueueArgs type as a parameter, the function here will now create the struct for the purposes of this initial request to get the bundle's metadata.